yarn.lock

```diff
@@ -18,6 +18,13 @@
   optionalDependencies:
```

Comment:
Seems like a lot of changes for one addition to package.json?

Reply:
Yeah it looks like a lot, seems the package I added has a lot of dev dependencies: https://github.com/Sambego/storybook-state/blob/master/package.json

I've had a look and it's the most popular for this purpose so I think we stay with it. Are we concerned at all with how big dev dependencies is getting?